classes/assets/StylesheetManager.php

```diff
@@ -42,15 +42,23 @@ class StylesheetManagerCore extends AbstractAssetManager
     protected function getDefaultList()
     {
         return [
-            'external' => [],
-            'inline' => [],
+            'external' => array(),
+            'inline' => array(),
         ];
     }
 
-    public function register($id, $relativePath, $media = self::DEFAULT_MEDIA, $priority = self::DEFAULT_PRIORITY, $inline = false)
-    {
-        if ($fullPath = $this->getFullPath($relativePath)) {
-            $this->add($id, $fullPath, $media, $priority, $inline);
+    public function register(
+        $id,
+        $relativePath,
+        $media = self::DEFAULT_MEDIA,
+        $priority = self::DEFAULT_PRIORITY,
```

Comment:
Theses class seems identical. Can we use an abstract class and rely on it for the `register` and `add` functions ?